community/tutorials.rst

```diff
@@ -19,6 +19,7 @@ Some tutorials mentioned below provide more advanced tutorials, e.g. on 3D or sh
 Video tutorials
 ---------------
 
+- `Brackeys <https://youtu.be/LOhfqjmasi0?si=dksdugZQdHhI9C0O>`_ (2D, GDScript)
```

Comment:
Should be moved down, alphabetical order

Reply:
thanks for the response let me fix that!